beacon-chain/db/kv/schema.go

```diff
@@ -14,11 +14,13 @@ var (
 	validatorsBucket         = []byte("validators")
 	stateBucket              = []byte("state")
 
-	// Key indices.
-	shardIdx      = []byte("shard")
-	parentRootIdx = []byte("parent-root")
-	startEpochIdx = []byte("start-epoch")
-	endEpochIdx   = []byte("end-epoch")
+	// Key indices buckets.
+	parentRootIndicesBucket = []byte("parent-root-indices")
+	blockSlotIndicesBucket  = []byte("block-slot-indices")
+	shardIndicesBucket      = []byte("shard-indices")
```

Comment:
what is a shard index? This is ambiguous as many data structures have a shard field